canonical-web-and-design/ubuntu.com
# Update /masters-conference page with virtual event

## Done

- Updated the /masters-conference page with the upcoming virtual event
- **NOTE - waiting on brightalk registration link**

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/masters-conference
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [copy doc](https://docs.google.com/document/d/1Z63eMCe9o4Onk2ezPFLPDTbI-uJezAOzI-aSpXmZ5lc/edit#heading=h.vcxakwtmfdmp)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2532

## Screenshots

![image](https://user-images.githubusercontent.com/441217/77663256-c7f78b80-6f74-11ea-9ad8-b9847dfcac74.png)
